client/components/forms/form-dropdown.js

```diff
@@ -7,6 +7,11 @@ class FormDropdown extends Component {
     const { controlId, ...field } = formGroup || {}
     const { className, children, ...props } = this.props
 
+    delete field.layout
+    delete field.error
+    delete field.touched
+    delete field.valid
```

Comment:
Yes, https://facebook.github.io/react/docs/higher-order-components.html#convention-pass-unrelated-props-through-to-the-wrapped-component